native-image-config/README.md

```diff
@@ -0,0 +1,46 @@
+# How to generate GraalVM native image configuration
+
+## Step 1: Generate native image configuration files
+
+First, generate the native image configuration files by running the tests annotated with
+`GenerateNativeImageTrace` with [`native-image-agent`](https://www.graalvm.org/latest/reference-manual/native-image/metadata/AutomaticMetadataCollection/).
+
+```
+$ ../gradlew nativeImageConfig
+```
+
+The generation process consists of the following 3 sub-steps:
+
+1. `processNativeImageTraces` task processes the native image traces generated by running the tests annotated 
+   with `@GenerateNativeImageTrace`. It generates the native image configuration at `build/step-1-process-native-image-traces`.
+2. `cleanupNativeImageConfig` task cleans up the native image configuration generated by `processNativeImageTraces`.
```

Comment:
`simplifyNativeImageConfig`?
Could you also fix this file as well? 😉

Reply:
Oops! :sweat_smile: